composer.json

```
@@ -10,10 +10,10 @@
     "type" : "phpcodesniffer-standard",
     "require": {
         "php": ">=5.4",
-        "behat/behat": "^2.5",
+        "behat/behat": "^3.6.1",
```

Behat can't be upgraded right now, that will require a rewrite of most of the testing infrastructure.
```suggestion
        "behat/behat": "^2.5",
```